ansible/playbook/tasks/roles/postgres/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+# This playbook contains tasks executed during PMM Server update.
+
+- name: Create buckup dir
+  file:
+    path: /srv/backup/postges
+    state: directory
+    owner: postgres
+    group: postgres
+    mode: '0700'
+
+- name: Get postgres encoding
+  postgresql_query:
+    db: pmm-managed
+    query: SHOW SERVER_ENCODING;
+  register: postgres_encoding
+  become: yes
+  become_user: postgres
+  become_method: su
+
+- debug:
+    var: postgres_encoding.query_result
+
+- name: change encoding pmm-managed database
```

Comment:
```suggestion
- name: Change encoding of pmm-managed database
```

Reply:
Let's update the devcontainer accordingly. https://github.com/percona/pmm-managed/blob/PMM-2.0/.devcontainer/setup.py#L77-L97